fabfile.py

```diff
@@ -900,14 +1030,16 @@ def check_and_create_database_user():
     with settings(warn_only=True):
         checkUser = venvcmd('assembl-pypsql -1 -u {user} -p {password} -n {host} "{command}"'.format(
             command="SELECT 1 FROM pg_roles WHERE rolname='%s'" % (env.db_user),
-            python=env.venvpath + "/bin/python", password=env.db_password,
+            python=join(env.venvpath, "bin", "python"), password=env.db_password,
             host=env.db_host, user=env.db_user, projectpath=env.projectpath))
     if checkUser.failed:
         print(yellow("User does not exist"))
-        run_db_command(as_venvcmd('assembl-pypsql -u {db_user} "{command}"'.format(
+        run_db_command(as_venvcmd('{python} {pypsql} -u {db_user} -n {host} "{command}"'.format(
```

Comment:
This still does not work: postgres user is not created and console output is the following.

```
User does not exist
[localhost] sudo: . /home/assembl_user/assembl/venv/bin/activate && /home/assembl_user/assembl/venv/bin/python /home/assembl_user/assembl/assembl/scripts/pypsql.py -u postgres -n localhost "CREATE USER assembl WITH CREATEDB ENCRYPTED PASSWORD 'assembl'; COMMIT;"
[localhost] out: /bin/bash: /root/.bash_profile: Permission denied
[localhost] out: /bin/bash: /home/assembl_user/assembl/venv/bin/activate: Permission denied
[localhost] out: 


Fatal error: sudo() received nonzero return code 1 while executing!

Requested: . /home/assembl_user/assembl/venv/bin/activate && /home/assembl_user/assembl/venv/bin/python /home/assembl_user/assembl/assembl/scripts/pypsql.py -u postgres -n localhost "CREATE USER assembl WITH CREATEDB ENCRYPTED PASSWORD 'assembl'; COMMIT;"
Executed: sudo -S -p 'sudo password:'  -u "postgres"  /bin/bash -l -c ". /home/assembl_user/assembl/venv/bin/activate && /home/assembl_user/assembl/venv/bin/python /home/assembl_user/assembl/assembl/scripts/pypsql.py -u postgres -n localhost \"CREATE USER assembl WITH CREATEDB ENCRYPTED PASSWORD 'assembl'; COMMIT;\""

Aborting.
Disconnecting from localhost... done.
```